parallaxinc/BlocklyProp-cdn
# Solo 12 new project not clearing old project

Two changes:
 - The UI warning linking to the client downloads was being disabled, but by using jQuery, it won't throw an error if it can't find it.  The ID for the link will need to be restored in the Solo and BlocklyProp blocklyc.html/jsp pages
 - The new project modal is now using the title ("New Project" vs. "Edit Project Details" to determine whether the project code currently in the editor should be cleared.

To test:
 - Open a project or start a new project and add code.
 - Click the "new project" link the top-left dropdown.
 - Fill in the info for a new project
 - Click "continue", toolbox and canvas should be clear and set correctly for the specified board
 - save, reopen, ensure project only contains new code and name/board type persisted.
